crates/release_plz_core/src/registry_packages.rs

```diff
@@ -61,7 +61,7 @@ pub fn get_registry_packages(
                 })
                 .collect(),
         ),
-        Option::None => {
+        None => {
```

Comment:
I qualify None due to a warning on my editor ([rust-analyzer](https://doc.rust-lang.org/rustc/?search=non_snake_case)) with rust 1.81 (on my editor)

> Variable `None` should have snake_case name, e.g. `none`

Reply:
weird since None is not a variable.